src/scripts/fix_workspace_info.py

```diff
@@ -0,0 +1,85 @@
+"""
+Fixes workspace info to do the following.
+1. Make sure the "narrative" metadata field contains an int that points to the narrative.
+2. Make sure the "narrative_nice_name" metadata field is correct.
+3. Make sure the "is_temporary" metadata field exists and is correct.
+4. Adds a count of the number of narrative cells.
+5. Does nothing at all if there's > 1 narrative in the workspace.
+Note that while this fetches the Narrative object, it doesn't modify it in any way.
+"""
+from biokbase.workspace.client import Workspace
+
+ws_url = "https://ci.kbase.us/services/workspace"
+
+def fix_workspace_info(ws_id, token, verbose=False):
```

Comment:
It'd be more useful if this took an admin token and used admin methods to fix things up (I don't think there's an alter_workspace_metadata admin fn yet, so the script would have to set and unset permissions)

Reply:
Sure, we can do that. I mostly left that aside, since I'm not sure the best way for it to run. I kinda assumed that there's some magic way to get a superuser token. Would it be hard to write a fn like that?

Reply:
You just have to get a token for a user that's a workspace admin (or we make you one), and then use the administrative interface: https://ci.kbase.us/services/ws/docs/administrationinterface.html